content/Products/OpenshiftMonitoring/telemetry.md

```diff
@@ -110,6 +110,8 @@ spec:
 
 Your `PrometheusRule` object(s) should be created by your operator with your `ServiceMonitor` and/or `PodMonitor` objects.
 
+Recording rule names should follow the [`level:metric:operations` pattern](https://prometheus.io/docs/practices/rules/).
```

Comment:
since we add the comment today :)
```suggestion
Recording rule names should follow the [`level:metric:operations` pattern](https://prometheus.io/docs/practices/rules/). When counting items, prefer the `:count` suffix over the `:total` suffix (the latter is used to signal an accumulating counter, for example a total count of requests).
```